index.html

```diff
@@ -196,6 +202,16 @@ <h3>HbbTV</h3>
           support does not work there.
         </p>
       </section>
+      <section>
+        <h3>BBC Subtitle Guidelines</h3>
+        <p>
+          The BBC in their guidelines for subtitles provide requirements for synchronisation
+          ([[BBC]], section 5).  In particular, the guidelines state: "The subtitles should match the pace of speaking as closely as possible. 
+          Ideally, when the speaker is in shot, your subtitles should not anticipate speech by more than 1.5 seconds or hang up on 
+          the screen for more than 1.5 seconds after speech has stopped."
```

Comment:
It is also worth quoting [§6.1-6.3](http://bbc.github.io/subtitle-guidelines/#Match-subtitles-to-shot) which states: 

> It is likely to be less tiring for the viewer if shot changes and subtitle changes occur at the same time. Many subtitles therefore start on the first frame of the shot and end on the last frame.
>
> 6.2 Maintain a minimum gap when mismatched
>
> If you have to let a subtitle hang over a shot change, do not remove it too soon after the cut. The duration of the overhang will depend on the content.
>
> 6.3 Avoid straddling shot changes
>
> Avoid creating subtitles that straddle a shot change (i.e. a subtitle that starts in the middle of shot one and ends in the middle of shot two). To do this, you may need to split a sentence at an appropriate point, or delay the start of a new sentence to coincide with the shot change.

If we are to meet these requirements then not only does the authoring tool need to support it (and several do) but the playback system must also honour the provided timings, on the assumption that they are frame accurate to 25fps.